components/automate-gateway/gateway/services.go

```diff
@@ -662,10 +665,21 @@ func (s *Server) authRequest(r *http.Request,
 	return authzResp.Err()
 }
 
-func logError(err error, resource, action string) {
-	log.WithFields(log.Fields{
```

Comment:
Did we move away from `WithFields`?

Reply:
Nope, just realised that the function hadn't been used 🤷‍♂